domains/khuathuykien.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Describe the use of this subdomain",
+  "repo": "https://github.com/kiensamson0000/khuathuykien-portfolio",
+  "owner": {
+    "username": "kiensamson0000",
+    "email": "kienkh99@gmail.com"
+  },
+  "record": {
+    "CNAME": "kiensamson0000.github.io"
```

Comment:
404 GitHub Page. No content.

Reply:
@DEV-DIBSTER Thank you, can you tell me in detail what the error is? I guess what you mean is that line number 9 of CNAME is incorrectly declared in the correct syntax?

Reply:
We can't see your website from your CNAME record. If you have your website hosted then you can send us a preview of your website